lib/spritely/options.rb

```diff
@@ -9,50 +9,52 @@ module Spritely
   # into the `spritely-map` function, this simplifies and de-duplicates the
   # passed option keys.
   #
-  #   $application-sprite: spritely-map("application/*.png",
-  #     $arrow-repeat: true,
-  #     $arrow-spacing: 10px,
-  #     $another-image-position: right,
-  #     $spacing: 5px
-  #   );
+  #   //= repeat arrow true
+  #   //= spacing arrow 10
+  #   //= position another-image right
+  #   //= spacing 5
   #
-  # The options passed in above will be easily accessible via an instance of
+  # The directives passed in above will be easily accessible via an instance of
   # this class.
   #
   #   options['arrow'] => {repeat: true, spacing: 10}
   #   options['another-image'] => {position: 'right', spacing: 5}
-  class Options < Struct.new(:hash)
+  class Options < Struct.new(:set)
```

Comment:
I wonder if this should be combined with the `Spritely::Sprockets::Preprocessor` class, to avoid transforming the directives into an set, and then into a hash of hashes.